### PR TITLE
Security advisories for vulnerabilities fixed in v24.0.1

### PR DESCRIPTION
**Pull request by darosior**
There was one security vulnerability fixed in v24.0.1.

Thanks to Pieter Wuille and Suhas Daftuar for their help in writing this up.

### CODE REVIEW

File: _posts/en/posts/2024-09-18-disclose-headers-oom.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+A protection against this DoS was implemented in Bitcoin Core PR
+[#25717](https://github.com/bitcoin/bitcoin/pull/25717), whereby the node will first verify a
+presented chain has enough work before committing to store it. With that, Bitcoin Core no longer
+relies on having checkpoints to protect against any known attacks.
+
+## Attribution
+
+Credit goes to David Jaenson for independently re-discovering the attack, estimating its cost and
+suggesting modifications.
```
Comment (murchandamus) [inline _posts/en/posts/2024-09-18-disclose-headers-oom.md:65]: From the timeline and above description, I’m wondering whether Braydon Fuller was also supposed to be mentioned here.

Comment (darosior) [reply _posts/en/posts/2024-09-18-disclose-headers-oom.md]: Ugh, good catch, i dropped the "and Braydon Fuller" part somehow. Fixed.

**Comment by instagibbs** [comment]
Looks like the last release of catchup? Month after 28.0 disclose up to 25.X?

**Comment by darosior** [comment]
There is one more, need to release vulns (if any) in 24.x that were fixed in 25.0.

**Review by murchandamus — COMMENTED** [review]
LGTM

File: _posts/en/posts/2024-09-18-disclose-headers-oom.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  checkpoint. This makes the role of checkpoints even more security-critical.
+* 2014-04-09 - Block 295000 is mined, which becomes the last Bitcoin Core checkpoint. The protection
+  offered by checkpoints against block spam starts eroding from this point on as hashrate costs
+  decrease.
+* 2015-02-16 - Bitcoin Core 0.10.0 is released, with headers-first synchronization. This weakens the
+  low-difficulty block spam attack to a block *header* spam attack.
+* 2017-03-08 - Bitcoin Core 0.14.0 is released, which disentangles the skipping of script validation
+  from checkpoints, leaving them only relevant for protecting against block header spam.
+* 2019-01-28 - David Jaenson reports this issue to the Bitcoin Core security mailing list.
+* 2019-09-18 - Braydon Fuller emails the Bitcoin Core security list with a paper titled "Bitcoin
```
Comment (pinheadmz) [inline _posts/en/posts/2024-09-18-disclose-headers-oom.md:83]: https://bcoin.io/papers/bitcoin-chain-expansion.pdf

If you feel like linking

Comment (darosior) [reply _posts/en/posts/2024-09-18-disclose-headers-oom.md]: Sure, had also linked the email already above.

**Review by dergoegge — APPROVED** [review]
ACK 0e68bf9e7e6082658ed8fb22f3beceb2bb520a0c

**Review by Zero-1729 — COMMENTED** [review]
crACK 0e68bf9e7e6082658ed8fb22f3beceb2bb520a0c

**Comment by achow101** [comment]
ACK 0e68bf9e7e6082658ed8fb22f3beceb2bb520a0c

**Pull request merged**